src/view/new-address.js

```diff
@@ -37,7 +37,7 @@ const styles = StyleSheet.create({
   },
 });
 
-const NewAddressView = ({ store, invoice }) => (
+const NewAddressView = ({ store, nav, invoice }) => (
```

Comment:
For me, the address itself didn't show up for this view:
![no_address_showing_err](https://user-images.githubusercontent.com/10647624/41494503-9375b14a-70c8-11e8-9840-521cd606910c.png)

Reply:
Created an issue: https://github.com/lightninglabs/lightning-app/issues/394